### ISSUE TITLE
Cleaner Implementation of DnD Basic Actions

### ISSUE THREAD

**Issue by Arkenny** [ZerdBG3/DnD-Rebalancing#90]
Change the name of the "Disengage" action to "Disengage/Dodge".
Make the "Disengage" basic action a spell container containing "Disengage" action and "Dodge" action.
Make the "Shove" basic action a spell container containing "Shove" and "Shove (Prone)".
Change the name of the "Dash" basic action to "Dash/Prone"
Make the "Dash" basic action a spell container containing "Dash", "Go Prone" and "Stand Up".
Make the "Main Hand Attack" basic action a spell container containing "Main hand Attack" and "Main hand Attack (Disarm)"

Implementation Notes:

"Go Prone" Basic Action
Cost: None
Duration: Until you use "Stand Up"
Effect: Disadvantage on Attacks, Ranged attacks against the creature have Disadvantage, Attacks within 5 feet of the creature
have Advantage.

"Stand Up" Basic Action
Cost: Half Movement
Duration: Instant

Notes: I hope this is helpful in streamlining the process of adding basic action into the game now and in the future.

**Comment by FelipeRenault** [comment]
I'm afraid that's not possible. When I implemented Shove (Prone), I couldn't include a spell container in the basic actions hotbar (Throw is an exception and it's hardcoded, I can't replicate its behavior not even when I add it as a basic spell outside the common actions). That's why I decided to make a separate spell container outside the left common hotbar and group all the new actions I implement on the same place to avoid cluttering. I'll check if that's still the case on Patch 5, but I don't think anything changed this last update.

The Main Hand Attack is even worse. That's the spell it's cast when you decide to left click an enemy. I can't even being to grasp the effects of making it a spell container would cause for the players and the AI
